### PR TITLE
augment expression operator dialog

### PR DESCRIPTION
**Pull request by ChristianBeilschmidt**

### CODE REVIEW

File: src/app/operators/dialogs/expression-operator/expression-operator.component.html [hunk 1]
```diff
@@ -1,54 +1,78 @@
+<style type="text/css">
+    .custom-unit {
+    }
+</style>
```
Comment (jdroenner) [inline src/app/operators/dialogs/expression-operator/expression-operator.component.html]: not needed -> remove this

File: src/app/operators/dialogs/expression-operator/expression-operator.component.html [hunk 2]
```diff
@@ -1,54 +1,78 @@
...
+            <p>For the expression, each raster is referred to an alphabetic character, e.g., A for the first raster.
+                It is possible to use basic operations like <code>+</code>, <code>-</code>, <code>*</code> and <code>/</code>, e.g., <code>A
+                    + B</code>.</p>
+            <p>It is necessary to specify the output datatype for the expression.
+                The list indicates the datatype of the input.
+                Furthermore, it is required to specify the range of the data in the form of min and max values.
+                The system will choice a appropriate NO DATA value automatically.</p>
+            <p>There is the option to specify an output unit for the expression result, e.g. <em>temperature</em> in <em>°C</em>.
+                This will be shown in the legend and automatically used in plots, etc.</p>
+            <p>Since the input rasters can be in differing projections, it is required to specify the output projection.</p>
```
Comment (jdroenner) [inline src/app/operators/dialogs/expression-operator/expression-operator.component.html]: Indicate that this is the projection used when applying the operator.

File: src/app/operators/dialogs/expression-operator/expression-operator.component.html (hunk 1 above)
Comment (ChristianBeilschmidt) [reply src/app/operators/dialogs/expression-operator/expression-operator.component.html]: 331b345

File: src/app/operators/dialogs/expression-operator/expression-operator.component.html (hunk 2 above)
Comment (ChristianBeilschmidt) [reply src/app/operators/dialogs/expression-operator/expression-operator.component.html]: 73fdbfd

**Pull request merged**